www/css/themes/main.css

```diff
@@ -1,7 +1,7 @@
 body {
```

Comment:
You can change the scrollbar color by modifying the `fill` property of the `blocklyScrollbarHandle` class

![image](https://user-images.githubusercontent.com/32238504/62150622-319c9880-b307-11e9-91ee-cd35238a70c4.png)

Reply:
false 
![image](https://user-images.githubusercontent.com/33164598/62152836-91bc1c00-b2c0-11e9-925b-517242b52653.png)
![image](https://user-images.githubusercontent.com/33164598/62152865-9da7de00-b2c0-11e9-9a21-fb973a55548a.png)

Reply:
wait i had an error. it works now 🤦‍♂